bin/replication.js

```diff
@@ -61,7 +54,10 @@ function _buildResourcePolicy(source, target) {
 }
 
 function _setupS3Client(transport, endpoint, profile) {
-    const credentials = new SharedIniFileCredentials({ profile });
+    const filename = process.env.AWS_SHARED_CREDENTIALS_FILE ||
+        `${os.homedir()}/.aws/credentials`;
+    const credentials = new SharedIniFileCredentials({ profile, filename });
```

Comment:
I think you can make `credentials` a global variable, that would avoid reconstructing it for different clients. Globals are usually fine to expose global configuration values.